### PR TITLE
Added Action Rule Profile - Additional Communities resource and other vendor files

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Added Action Rule Profile - Additional Communities resource and other vendor files, Issue belongs to: https://github.com/CiscoDevNet/terraform-provider-aci/issues/826

### CODE REVIEW

File: aci/resource_aci_rtctrlsetaddcomm.go [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+		rtctrlSetAddCommAttr.Name = Name.(string)
+	}
+
+	if SetCriteria, ok := d.GetOk("set_criteria"); ok {
+		rtctrlSetAddCommAttr.SetCriteria = SetCriteria.(string)
+	}
+
+	if Type, ok := d.GetOk("type"); ok {
+		rtctrlSetAddCommAttr.Type = Type.(string)
+	}
```
Comment (lhercot) [inline aci/resource_aci_rtctrlsetaddcomm.go:158]: not needed

File: aci/resource_aci_rtctrlsetaddcomm.go [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+		rtctrlSetAddCommAttr.Annotation = "{}"
+	}
+
+	if Community, ok := d.GetOk("community"); ok {
+		rtctrlSetAddCommAttr.Community = Community.(string)
+	}
+
+	if Name, ok := d.GetOk("name"); ok {
+		rtctrlSetAddCommAttr.Name = Name.(string)
+	}
```
Comment (lhercot) [inline aci/resource_aci_rtctrlsetaddcomm.go:197]: not needed

File: examples/action_rule_profile_additional_communities/main.tf [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+resource "aci_action_rule_profile" "foo_action_rule_profile" {
+  name        = "tf_rule"
+  description = "action_rule_profile created while acceptance testing"
+  tenant_dn   = aci_tenant.foo_tenant.id
+}
+
+resource "aci_action_rule_profile_additional_communities" "foo_rtctrl_set_add_comm" {
+  community              = "no-advertise"
```
Comment (lhercot) [inline examples/action_rule_profile_additional_communities/main.tf:28]: put the example to the same as proposed in the UI.

File: vendor/modules.txt [hunk 4]
```diff
@@ -54,7 +54,7 @@ github.com/aws/aws-sdk-go/service/sts
 github.com/aws/aws-sdk-go/service/sts/stsiface
 # github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d
 github.com/bgentry/go-netrc/netrc
-# github.com/ciscoecosystem/aci-go-client v1.30.0
+# github.com/ciscoecosystem/aci-go-client v1.32.0 => /Users/sajagana/go/src/github.com/ciscoecosystem/aci-go-client
```
Comment (lhercot) [inline vendor/modules.txt:57]: Remove the replace before committing.

Comment (lhercot) [reply vendor/modules.txt:57]: And make sure to do the go get ... and go mod vendor

File: website/docs/d/action_rule_profile_additional_communities.html.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+* `Location` - Tenant > Policies > Protocols > Set Rules -> Rule -> Additional Communities
+
+
+## Example Usage ##
+
+```hcl
+data "aci_action_rule_profile_additional_communities" "example" {
+  action_rule_profile_dn  = aci_action_rule_profile.example.id
+  community  = "example"
```
Comment (lhercot) [inline website/docs/d/action_rule_profile_additional_communities.html.markdown:30]: change example to actual proposed example in UI.

File: website/docs/d/action_rule_profile_additional_communities.html.markdown [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+* `action_rule_profile_dn` - (Required) Distinguished name of the parent action rule profile object.
+* `community` - (Required) The community value of the set action rule profile additional communities object.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the additional communities object.
+* `annotation` - (Optional) Annotation of the additional communities object.
+* `name_alias` - (Optional) Name Alias of the additional communities object.
+* `set_criteria` - (Optional) The criteria for setting the (extended) community attribute for a BGP route update.
+* `type` - (Optional) The type of the set action rule profile additional communities object.
```
Comment (lhercot) [inline website/docs/d/action_rule_profile_additional_communities.html.markdown:44]: remove not needed

File: website/docs/r/action_rule_profile_additional_communities.html.markdown [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## Example Usage ##
+
+```hcl
+resource "aci_action_rule_profile_additional_communities" "example" {
+  action_rule_profile_dn  = aci_action_rule_profile.example.id
+  community  = "example"
+  annotation = "orchestrator:terraform"
+  set_criteria = "append"
+  type = "community"
```
Comment (lhercot) [inline website/docs/r/action_rule_profile_additional_communities.html.markdown:33]: remove type

File: website/docs/r/action_rule_profile_additional_communities.html.markdown [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+* `Location` - Tenant > Policies > Protocols > Set Rules -> Rule -> Additional Communities
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_action_rule_profile_additional_communities" "example" {
+  action_rule_profile_dn  = aci_action_rule_profile.example.id
+  community  = "example"
```
Comment (lhercot) [inline website/docs/r/action_rule_profile_additional_communities.html.markdown:30]: Use the same community as shown as example in the UI

File: website/docs/r/action_rule_profile_additional_communities.html.markdown [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+## Argument Reference ##
+
+* `action_rule_profile_dn` - (Required) Distinguished name of the parent action rule profile object.
+* `community` - (Required) The community value of the set action rule profile additional communities object.
+* `annotation` - (Optional) Annotation of the action rule profile additional communities object.
+* `name_alias` - (Optional) Name Alias of the additional communities object.
+* `set_criteria` - (Optional) The criteria for setting the (extended) community attribute for a BGP route update. Allowed values are "append", "none", "replace", and default value is "append". Type: String.
+* `type` - (Optional) Allowed values are "as-path", "community", "dampening-pol", "ip-nh", "local-pref", "metric", "metric-type", "nh-unchanged", "ospf-fwd-addr", "ospf-nssa", "redist-multipath", "rt-tag", "rt-weight", and default value is "community". Type: String.
```
Comment (lhercot) [inline website/docs/r/action_rule_profile_additional_communities.html.markdown:44]: remove type

**Review by lhercot — CHANGES_REQUESTED** [review]

File: examples/action_rule_profile_additional_communities/main.tf (hunk 3 above)
Comment (sajagana) [reply examples/action_rule_profile_additional_communities/main.tf:28]: UI does not have the placeholder value and "no-advertise" is one of the options in UI.

File: website/docs/d/action_rule_profile_additional_communities.html.markdown [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+## Argument Reference ##
+
+* `action_rule_profile_dn` - (Required) Distinguished name of the parent action rule profile object.
+* `community` - (Required) The community value of the set action rule profile additional communities object.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the additional communities object.
+* `annotation` - (Optional) Annotation of the additional communities object.
```
Comment (lhercot) [inline website/docs/d/action_rule_profile_additional_communities.html.markdown:41]: I think description is missing here.

File: website/docs/r/action_rule_profile_additional_communities.html.markdown [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+  community  = "no-advertise"
+  annotation = "orchestrator:terraform"
+  set_criteria = "append"
+}
+```
+
+## Argument Reference ##
+
+* `action_rule_profile_dn` - (Required) Distinguished name of the parent action rule profile object.
+* `community` - (Required) The community value of the set action rule profile additional communities object.
```
Comment (lhercot) [inline website/docs/r/action_rule_profile_additional_communities.html.markdown:39]: I think description is missing here.

**Review by lhercot — CHANGES_REQUESTED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**